### PR TITLE
Add URL to Wikia Extensions and $wgExtensionCredits to some that were missing it (3)

### PR DESCRIPTION
**Pull request by nandy-andy**
The original pull requests are found at #5745 and #5860.

First one was closed and reverted due to breaking the `dev` branch. @ty-a have fixed the syntax errors.
Second one was closed because it couldn't be merged by GitHub and I've resolved the conflicts manually.

I smoke tested it on my devbox and ran unit tests against it.

@ty-a can you take a look and check if all your changes are here?
@timmyquivy you might want to double check it :cat:

### CODE REVIEW

**Comment by nandy-andy** [comment]
Oh, I can see now only 55 files changed. Something's wrong. We've got 395 files changed in #5745 and #5860 :-1: Let me take a look at it a little bit longer...

File: extensions/wikia/AdminDashboard/AdminDashboard.i18n.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 $messages = array();
 
 $messages['en'] = array(
+	'admindashboard-desc' => '{{desc}}',
```
Comment (Grunny) [inline extensions/wikia/AdminDashboard/AdminDashboard.i18n.php]: This is a `qqq` string, needs the actual description message.

File: extensions/wikia/AnalyticsEngine/AnalyticsEngine.setup.php [hunk 2]
```diff
@@ -21,6 +26,9 @@
 $wgAutoloadClasses['AnalyticsProviderPageFair'] = __DIR__ . '/AnalyticsProviderPageFair.php';
 $wgAutoloadClasses['AnalyticsProviderRubiconRTP'] = __DIR__ . '/AnalyticsProviderRubiconRTP.php';
 
+//i18n
+$wgExtensionMessagesFiles['AnalyticsEngine'] = __DIR__ . '/i18n/AnalyticsEngine.i18n.php';
```
Comment (Grunny) [inline extensions/wikia/AnalyticsEngine/AnalyticsEngine.setup.php]: This has been added twice to this file.

**Comment by ty-a** [comment]
It looks like all of my changes are here.

**Comment by nandy-andy** [comment]
I'll wait for @Grunny to review it, I'll fix the small issues and hopefully we'll merge it sooner than later :japanese_ogre:

File: extensions/wikia/AdminDashboard/AdminDashboard.i18n.php (hunk 1 above)
Comment (nandy-andy) [inline extensions/wikia/AdminDashboard/AdminDashboard.i18n.php]: Isn't it an element of English messages?

**Comment by nandy-andy** [comment]
@timmyquivy, @Grunny are you fine with merging it today? I've merged `dev` again, all unit tests are passing and I've double checked it. 

If we merge it today it'll "stay" for a while in `dev` before hiting production. So, if anything completely breaking got there Engineering will notice :smile_cat:

File: extensions/wikia/ApiDocs/i18n/ApiDocs.i18n.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+$messages = array();
+
+$messages['en'] = array(
+	'apidocs-desc' => 'Api Documentation',
```
Comment (Grunny) [inline extensions/wikia/ApiDocs/i18n/ApiDocs.i18n.php]: Should probably switch "Api" -> "API".

File: extensions/wikia/AssetsManager/AssetsManager_setup.php [hunk 4]
```diff
@@ -12,25 +12,32 @@
...
+$wgAutoloadClasses['AssetsManagerException'] = __DIR__ . '/AssetsManagerException.class.php';
+
+$wgAutoloadClasses['AssetsManagerController'] = __DIR__ . '/AssetsManagerController.class.php';
 
 $wgAjaxExportList[] = 'AssetsManagerEntryPoint';
 $wgHooks['MakeGlobalVariablesScript'][] = 'AssetsManager::onMakeGlobalVariablesScript';
 $wgHooks['UserGetRights'][] = 'onUserGetRights';
 
+
+
```
Comment (Grunny) [inline extensions/wikia/AssetsManager/AssetsManager_setup.php]: Unnecessary extra white space.

File: extensions/wikia/Forum/ForumDisabled.setup.php [hunk 5]
```diff
@@ -8,7 +8,9 @@
 
 $wgExtensionCredits['specialpage'][] = array( 
 	'name' => 'Forum',
-	'author' => array( 'Kyle Florence', 'Saipetch Kongkatong', 'Tomasz Odrobny' )
+	'author' => array( 'Kyle Florence', 'Saipetch Kongkatong', 'Tomasz Odrobny' ),
+	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia',
```
Comment (Grunny) [inline extensions/wikia/Forum/ForumDisabled.setup.php]: I imagine this should also be `https://github.com/Wikia/app/tree/dev/extensions/wikia/Forum`.

**Comment by nandy-andy** [comment]
Discussed with @Grunny on Slack. Addressed recent issues and merged it.

**Pull request merged**